test/config/updatedeletetest.go

```diff
@@ -17,6 +17,7 @@ package config
 import (
 	gnmiutils "github.com/onosproject/onos-config/test/utils/gnmi"
 	"github.com/onosproject/onos-config/test/utils/proto"
+	gbp "github.com/openconfig/gnmi/proto/gnmi"
```

Comment:
we should try to use the same alias for this gnmi thing. I guess gnmiapi should be good.